x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/constants.ts

```diff
@@ -51,7 +51,6 @@ export const DEFINITION_UPGRADE_FIELD_ORDER: Array<keyof DiffableAllFields> = [
   'threat_indicator_path',
   'concurrent_searches',
   'items_per_search',
```

Comment:
Shouldn't these be hidden in the UI as well?

Reply:
Sorry, it looks like I missed updating the table for these two, but when reworking on https://github.com/elastic/kibana/issues/180393 my decision was:

- keep both fields in the schema
- keep both fields in the diff calculation
- show them in the diffing UI, but only as read-only (same as `author` and `license`)

The idea being that it's valuable to inform the user if Elastic is proposing a change in the rule, but not allowing the user to modify it. Again, same behaviour as `author` and `license`.

Reply:
> show them in the diffing UI, but only as read-only (same as author and license)

So, how we'd handle the ABC scenario for these fields if they are read-only in the UI?

Reply:
> So, how we'd handle the ABC scenario for these fields if they are read-only in the UI?

Yes, that's my idea.

Reply:
Discussed over Zoom, will create new difiing algorithms for these two fields: ticket link.